main.js

```diff
@@ -210,17 +210,16 @@ function getAllRepos() {
 
     const ss = SpreadsheetApp.getActiveSpreadsheet();
     const sheet = ss.getSheetByName(`プルリク情報`);
-    for (; i < pullRequests.length; i++) {
-      const pullRequest = pullRequests[i];
-  
+    for (const pullRequest of pullRequests.length) {
```

Comment:
```suggestion
    for (const pullRequest of pullRequests) {
```
こちら正しくは👆ではないでしょうか？

PR頂いたコードを実行したところ👇のエラーが出ております。
![スクリーンショット 2024-07-03 6 10 30](https://github.com/cosoji-jp/github-gas-four-keys/assets/39693776/b9be52de-c483-4d92-96f5-0a690bcd2385)

参考資料: [for...ofのMSDNリファレンス](https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Statements/for...of)

Reply:
通知に気づかず失礼いたしました🙇‍♂️
修正ありがとうございます